src/util/countryCallingCodes.js

```diff
@@ -1,13 +1,253 @@
-/*!
```

Comment:
the scripts still have defect (didn't add copyright notes). would you mind add copyright manually after file has been created?

Reply:
+1 to add back copyright notes

Reply:
fixed